safekeeper/src/remove_wal.rs

```diff
@@ -2,21 +2,26 @@ use utils::lsn::Lsn;
 
 use crate::timeline_manager::StateSnapshot;
 
-/// Get oldest LSN we still need to keep. We hold WAL till it is consumed
-/// by all of 1) pageserver (remote_consistent_lsn) 2) peers 3) s3
-/// offloading.
-/// While it is safe to use inmem values for determining horizon,
-/// we use persistent to make possible normal states less surprising.
-/// All segments covering LSNs before horizon_lsn can be removed.
+/// Get oldest LSN we still need to keep. We hold WAL till it is consumed by 1)
```

Comment:
nit: Could add a paragraph break here with the sentence to have a nice summary and rest, and the rest could start with ordered list instead of inline for readability (inline lists do not turn into lists in html).

Reply:
done